x/tx/signing/textual/types.go

```diff
@@ -34,10 +34,12 @@ type Screen struct {
 // here, so that optionally more value renderers could be built, for example, a
 // separate one for a different language.
 type ValueRenderer interface {
-	// Format should render the value to a text plus annotation.
+	// Format renders the Protobuf value to a list of Screens.
 	Format(context.Context, protoreflect.Value) ([]Screen, error)
 
-	// Parse should be the inverse of Format.
+	// Parse is the inverse of Format. It must be able to parse all valid
+	// screens, however the behavior of Parse on invalid screens is not
```

Comment:
Clarify that "invalid" means "not produced by this renderer's Format() method". Here and below.